test/project/pkg/webhook/default_server/server.go

```diff
@@ -46,6 +46,9 @@ func Add(mgr manager.Manager) error {
 		secretName = "webhook-server-secret"
 	}
 
+	// +kubebuilder:webhook:port=7890,cert-dir=/tmp/test-cert,service=test-system:webhook-service,selector=app:webhook-server
```

Comment:
We should probably add a comment just above these annotations with a link to some page on kubebuilder book which explains these ?